redaxo/src/core/lib/util/i18n.php

```diff
@@ -142,7 +137,15 @@ private static function getMsg($key, $htmlspecialchars, array $args)
                 $replacements[] = $args[$i];
             }
         }
-        return preg_replace($patterns, $replacements, $msg);
+
+        $msg = preg_replace($patterns, $replacements, $msg);
+
+        if ($htmlspecialchars) {
+            $msg = htmlspecialchars($msg);
```

Comment:
Muss hier wie in rex-escape auch mit zeichensatz etc gearbeitet werden (parameter)